hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -80,6 +80,10 @@ public Operation() {
         setFlag(true, BITMASK_CALL_TIMEOUT_64_BIT);
     }
 
+    public boolean executedLocally() {
```

Comment:
Aren't backups in a one-node cluster executed on the same node? Will this solution work for that scenario?